spec/controllers/search_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
```

Comment:
This might be on your radar already (if so, ignore):
It'd be good to have the tests also look for some things on the page that indicate that it loaded correctly. For example, a facility/service shows a few pieces of correct information.